design/architecture.excalidraw

```diff
@@ -0,0 +1,3270 @@
+{
```

Comment:
Is this the source for arch.png below?  Maybe rename them to share a basename without suffix?

Reply:
Moved - the didn't kept them at the same place first because didn't thought we wanted the source distributed with the docs. But why not :)